src/github/pullRequestManager.ts

```diff
@@ -1044,13 +1052,13 @@ export class PullRequestManager implements vscode.Disposable {
 		return inDraftMode;
 	}
 
-	async updateDraftModeContext(pullRequest: PullRequestModel) {
-		if (this._activePullRequest && this._activePullRequest.prNumber === pullRequest.prNumber) {
+	async updateDraftModeContext(pullRequest: IssueModel) {
+		if (this._activeGithubItem && this._activeGithubItem.githubNumber === pullRequest.githubNumber) {
 			await vscode.commands.executeCommand('setContext', 'reviewInDraftMode', pullRequest.inDraftMode);
 		}
 	}
 
-	async getPendingReviewId(pullRequest = this._activePullRequest): Promise<string | undefined> {
+	async getPendingReviewId(pullRequest = this._activeGithubItem): Promise<string | undefined> {

```

Comment:
same here, since you can't add reviews to issues, I think this should remain `this._activePullRequest`

Reply:
Changed